code/modules/clothing/head/helmet.dm

```diff
@@ -142,7 +142,7 @@
 	name = "gladiator helmet"
 	desc = "Ave, Imperator, morituri te salutant."
 	icon_state = "gladiator"
-	flags = HEADCOVERSEYES|BLOCKHAIR
+	flags = HEADCOVERSEYES|BLOCKHAIR|HEADCOVERSMOUTH

```

Comment:
http://www.mullocksauctions.co.uk/wmsimages/TM-18Oct2011G/815.jpg

I could get a donk pocket through that mouth slot.